src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -82,4 +126,15 @@ public synchronized Optional<HTTPServer> startHttpServer() {
         }
     }
 
+    static class ListenerValidator implements ConfigDef.Validator {
+
+        @Override
+        public void ensureValid(String name, Object value) {
+            String str = (String) value;
+            Matcher matcher = Listener.PATTERN.matcher(str);
+            if (!matcher.matches()) {
+                throw new ConfigException(value + " is an invalid listener format. Must be of format http:// [host]:[port] ");
```

Comment:
Can we use the constructor with 3 arguments?
https://kafka.apache.org/36/javadoc/org/apache/kafka/common/config/ConfigException.html#%3Cinit%3E(java.lang.String,java.lang.Object,java.lang.String)

Reply:
Do you mean like this?:
```throw new ConfigException("Invalid listener format: " + value + ". Must be of format http:// [host]:[port]");```

Reply:
I mean something like `throw new ConfigException(name, value, "Listener must be of format http:// [host]:[port]");`
Same above

Reply:
Done

Reply:
You don't seem to have addressed this. The code is still using the constructor with a single argument.